packages/orchestration/src/facade.js

```diff
@@ -153,15 +157,19 @@ const makeRemoteChainFacade = (
  *   storageNode: Remote<StorageNode>;
  *   orchestrationService: Remote<OrchestrationService>;
  *   localchain: Remote<LocalChain>;
+ *   agoricNames: Remote<NameHub>;
  * }} powers
  */
 export const makeOrchestrationFacade = ({
+  // TODO zone is a lot of power to hand down. If it's only used for
+  // `prepareCosmosOrchestrationAccount` we can do that in the parent scope
```

Comment:
IIUC, making an async flow needs a durable zone.
I expect it can be a sub-zone and need not be the whole thing.

Reply:
ah yes. I'll remote this comment.